packages/nux/src/components/dot-tip/index.js

```diff
@@ -57,17 +57,15 @@ export function DotTip( {
 			<p>{ children }</p>
 			<p>
 				<Button
-					// TODO: Switch to `true` (40px size) if possible
-					__next40pxDefaultSize={ false }
+					__next40pxDefaultSize
```

Comment:
The `nux` package is not used anymore and has long been deprecated so we can't see an actual render, but here is an old screenshot:

![Old screenshot of nux dot-tip](https://github.com/user-attachments/assets/e533c280-e09e-4682-8213-2d127d39af8d)

The link variant should remain the same height, and the "Disable tips" button is the close button in the top right.